src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1493,7 +1493,7 @@ static unsigned uct_dc_mlx5_ep_fc_hard_req_progress(void *arg)
      * packet in case of failure on the remote FC endpoint */
     kh_foreach_key(&iface->tx.fc_hash, ep_key, {
         ep = (uct_dc_mlx5_ep_t*)ep_key;
-        uct_dc_mlx5_ep_schedule(iface, ep);
+        uct_dc_mlx5_ep_schedule(iface, ep, 1);
```

Comment:
maybe add a comment like 
"Force DCI scheduling, since FC resources may never become available unless we send this request"

Reply:
yes, makes sense
done